tests/collections/common/tokenizers/text_to_speech/test_tts_tokenizers.py

```diff
@@ -252,3 +257,15 @@ def test_ipa_tokenizer_fixed_vocab(self):
         chars, tokens = self._parse_text(tokenizer, "Hello, wound")
         expected_output = "HELLO, ˈwund"
         assert chars == expected_output
+
+    @pytest.mark.run_only_on('CPU')
+    @pytest.mark.unit
+    def test_ipa_tokenizer_ja_jp(self):
+        input_text = "ハロー ワールド."
+        expected_output = "haɾoː wa:ɾdo."
+        g2p = IpaG2p(phoneme_dict=self.PHONEME_DICT_EN)
```

Comment:
this unit test is not expected since your recipe applied `JapaneseG2p` rather than `IpaG2p`. In this file, pls add `test_japanese_phoneme_tokenizer(self)` and choose `JapanesePhonemeTokenizer`.

Reply:
Added <nemo.collections.tts.g2p.models.ja_jp_g2p.py import JapaneseG2> to update line 267. 
    @pytest.mark.run_only_on('CPU')
    @pytest.mark.unit
    def test_japanese_phoneme_tokenizer(self):
        input_text = "ハロー ワールド."
        expected_output = "haɾoː wa:ɾdo."
        g2p = JapaneseG2p(phoneme_dict=self.PHONEME_DICT_JA)

        tokenizer = JapanesePhonemeTokenizer(g2p=g2p)
        chars, tokens = self._parse_text(tokenizer, input_text)

        assert chars == expected_output